tests/c21client/test_client.py

```diff
@@ -1,44 +1,38 @@
-import os
-import pytest
 import requests
 from c21client.client import Client
 
 BASE_URL = "http://localhost:8080"
 
 
-@pytest.fixture
-def setup():
-    os.remove('client.cfg')
-
-
 def test_client_url():
     client = Client()
     assert client.url == BASE_URL
 
 
-def test_client_gets_existing_client_id():
+def test_client_gets_existing_client_id(mocker):
     client = Client()
     mock_client_id = 99
-    path = 'client.cfg'
-    with open(path, "w") as file:
-        file.write(str(mock_client_id))
+    write_mock_client_id(mocker)
+    read_mock_client_id(mocker, mock_client_id)
     assert mock_client_id == client.get_client_id()
 
 
-def test_client_gets_client_id_from_server(requests_mock):
+def test_client_gets_client_id_from_server(requests_mock, mocker):
     client = Client()
-    os.remove('client.cfg')
     mock_client_id = 9
+    read_mock_client_id(mocker, -1)
+    write_mock_client_id(mocker)
     requests_mock.get(
```

Comment:
Need to re-add pytest fixture; code fails on this line with "fixture 'requests_mock' not found"

Reply:
Could you tell me the steps you are taking to run the tests so I can try to reproduce the error? I don't get any errors.

Reply:
Nevermind, looks like it was a problem on my end. PR looks good.